Yosemite/Yosemite/PointOfSale/POSItemProvider.swift

```diff
@@ -3,6 +3,7 @@ public protocol POSItem {
     var productID: Int64 { get }
     var name: String { get }
     var price: String { get }
+    var formattedPrice: String { get }
```

Comment:
Added formattedPrice because clean price is expected when creating/syncing the order to get a proper totals calculation

Reply:
formatted price is a price with the currency symbol.. because the price coming from backend is without the currency symbol. So to avoid doing the currency formatting all the time I just added this to be available

Reply:
Sure, that makes sense. 

My question was about why we still need `price` though? I didn't think we were going to do any calculations in the app for MVP...